src/runtime/auto-prompt-manager.js

```diff
@@ -187,6 +189,15 @@ export class AutoPromptManager {
     dismissedPrompts,
     params
   ) {
+    const audienceActionsPromptType = article
+      ? this.getAutoPromptType_(article)
+      : undefined;
+
+    // Override autoPromptType if it is undefined.
+    params.autoPromptType = params.autoPromptType
```

Comment:
This would be a little simpler
```suggestion
    params.autoPromptType = params.autoPromptType || audienceActionsPromptType;
```

Reply:
Thanks for the suggestion!